notify/receivers_test.go

```diff
@@ -68,19 +57,19 @@ func (e timeoutError) Timeout() bool {
 }
 
 func TestProcessNotifierError(t *testing.T) {
-	t.Run("assert ReceiverTimeoutError is returned for context deadline exceeded", func(t *testing.T) {
-		r := &GrafanaReceiver{
+	t.Run("assert IntegrationTimeoutError is returned for context deadline exceeded", func(t *testing.T) {
```

Comment:
I'm a bit confused, is there some behavioural change here too?

Reply:
https://github.com/grafana/alerting/pull/73#discussion_r1152380431

Reply:
Not sure I understand what you mean. I do not see any changes in this test other than names

Reply:
Again this now looks different from when I commented, GitHub being weird?